lib/rubocop/cop/rspec/factory_bot/dynamic_attribute_defined_statically.rb

```diff
@@ -31,13 +31,13 @@ class DynamicAttributeDefinedStatically < Cop
             (block (send nil? {:factory :trait} ...) _ { (begin $...) $(send ...) } )
           PATTERN
 
-          def_node_matcher :callback_with_symbol_proc?, <<-PATTERN
-            (send nil? {:before :after} sym (block_pass sym))
+          def_node_matcher :callback_or_sequence_with_symbol_proc?, <<-PATTERN
```

Comment:
I am thinking this should perhaps be two different methods? I.e. adding a new `sequence_with_symbol_proc` node matcher. What do you think?